packages/babel-preset/common.js

```diff
@@ -76,6 +76,11 @@ module.exports = function shopifyCommonPreset(
       require.resolve('@babel/plugin-proposal-class-properties'),
       {loose: true},
     ],
+    // The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods
```

Comment:
So I mean this to mean that because `@babel/plugin-proposal-class-properties` is set to `loose: true` due to typescript then the `@babel/plugin-proposal-private-methods` must also be true.

Reply:
Mostly so I don't forget why I added this.